reactrestaurant/src/components/navtabs.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Badge from './badge';
+
+function NavTabs() {
+    return (
+        <>
+            <ul className="nav nav-pills mb-3 justify-content-center" id="pills-tab" role="tablist">
+                <li className="nav-item">
+                    <a className="nav-link active" id="pills-specials-tab" data-toggle="pill" href="#pills-specials" role="tab" aria-controls="pills-specials" aria-selected="true">Specials <Badge /></a>
+                </li>
+                <li className="nav-item">
+                    <a className="nav-link" id="pills-breakfast-tab" data-toggle="pill" href="#pills-breakfast" role="tab" aria-controls="pills-breakfast" aria-selected="false">Breakfast <Badge /></a>
+                </li>
+                <li className="nav-item">
+                    <a className="nav-link" id="pills-lunch-tab" data-toggle="pill" href="#pills-lunch" role="tab" aria-controls="pills-lunch" aria-selected="false">Lunch <Badge /></a>
+                </li>
+                <li className="nav-item">
+                    <a className="nav-link" id="pills-dinner-tab" data-toggle="pill" href="#pills-dinner" role="tab" aria-controls="pills-dinner" aria-selected="false">Dinner <Badge /></a>
+                </li>
+                <li className="nav-item">
+                    <a className="nav-link" id="pills-bar-tab" data-toggle="pill" href="#pills-bar" role="tab" aria-controls="pills-bar" aria-selected="false">Bar <Badge /></a>
```

Comment:
Here is where you could pass the count into your badge component something like 
`<a href="#" ><Badge count{itemCount} />`

and then in Badge.js you could receive it as a prop
```
function Badge(props) {
            return <div className="badge">{props.count}</div>;
}
```